core/integration/features/verified/0011-MARA-006.feature

```diff
@@ -98,3 +97,298 @@ Feature: check margin account with partially filled order
+    And the mark price should be "10" for the market "ETH/DEC20"
+    When the parties place the following orders with ticks:
+      | party    | market id | side | volume | price | resulting trades | type       | tif     | reference   |
+      | trader2  | ETH/DEC20 | sell | 40     | 50    | 0                | TYPE_LIMIT | TIF_GFN | buy-order-3 |
+      | trader20 | ETH/DEC20 | buy  | 40     | 50    | 1                | TYPE_LIMIT | TIF_GFN | buy-order-3 |
+
+    And the parties should have the following margin levels:
+      | party   | market id | maintenance | search | initial | release |
+      | trader2 | ETH/DEC20 | 45234       | 54280  | 67851   | 90468   |
+
+    Then the parties should have the following account balances:
+      | party   | asset | market id | margin | general |
+      | trader2 | USD   | ETH/DEC20 | 67851  | 22149   |
+
+    When the parties place the following orders with ticks:
+      | party   | market id | side | volume | price | resulting trades | type       | tif     | reference   |
+      | trader2 | ETH/DEC20 | buy  | 40     | 50    | 0                | TYPE_LIMIT | TIF_GFN | buy-order-4 |
+
+    And the parties should have the following margin levels:
+      | party   | market id | maintenance | search | initial | release |
+      | trader2 | ETH/DEC20 | 45234       | 54280  | 67851   | 90468   |
+
+    Then the parties should have the following account balances:
+      | party   | asset | market id | margin | general |
+      | trader2 | USD   | ETH/DEC20 | 67851  | 22149   |
+
+    When the parties place the following orders with ticks:
+      | party    | market id | side | volume | price | resulting trades | type       | tif     | reference    |
+      | trader20 | ETH/DEC20 | sell | 10     | 50    | 1                | TYPE_LIMIT | TIF_GFN | sell-order-4 |
+
+    And the parties should have the following margin levels:
+      | party   | market id | maintenance | search | initial | release |
+      | trader2 | ETH/DEC20 | 34826       | 41791  | 52239   | 69652   |
+
+    Then the parties should have the following account balances:
+      | party   | asset | market id | margin | general |
+      | trader2 | USD   | ETH/DEC20 | 67851  | 22149   |
+
+    When the parties place the following orders with ticks:
+      | party    | market id | side | volume | price | resulting trades | type       | tif     | reference    |
+      | trader20 | ETH/DEC20 | sell | 1      | 50    | 1                | TYPE_LIMIT | TIF_GFN | sell-order-4 |
+
+    And the parties should have the following margin levels:
+      | party   | market id | maintenance | search | initial | release |
+      | trader2 | ETH/DEC20 | 33636       | 40363  | 50454   | 67272   |
+
+    Then the parties should have the following account balances:
+      | party   | asset | market id | margin | general |
+      | trader2 | USD   | ETH/DEC20 | 50454  | 39546   |
+
+Scenario: 004 check margin for GTT order type.0011-MARA-007
```

Comment:
Is this scenario really necessary? I don't see much value over scenario 002. Sure, the GTT order in one market expires and the GTC one in another market doesn't, but does it really give us much?